index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/045dcd1183.js" crossorigin="anonymous"></script>
+  <title>Jeff Kayombo- Portfolio</title>
+</head>
+<body>
+  <header class="tool-bar">
+    <div class="logo">
+      <h2><a href="#">Welcome</a></h2>
+    </div>
+    <nav>
+      <i class="fas fa-bars"></i>
```

Comment:
- [ ] [OPTIONAL] I think it can be a good idea to have this icon inside of an element that is clickable since we are supposed to click on the hamburger icon to view the links, so having to wrap the icon inside a `<button>` or `<a>` tag could make more sense 😉